nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/graphrunner/GraphRunner.java

```diff
@@ -435,27 +308,30 @@ public GraphRunner(SavedModelConfig savedModelConfig, ConfigProto sessionOptions
             //the result pointers are null, and will be populated automatically by the session run
             PointerPointer<TF_Tensor> outputTensorsPointer = new PointerPointer<>(savedModelConfig.getSaveModelOutputOrder().size());
 
-
+            long start = System.nanoTime();
             TF_SessionRun(
                     session,
                     null,
                     //inputs
                     inputOut, inputTensorsPointer, inputTensors.length,
-                    //outputs
+                    //outputSchema
                     outputOut, outputTensorsPointer, savedModelConfig.getSaveModelOutputOrder().size(),
                     //targets
                     null, 0,
                     null,
-                    status);
+                    status);        long end = System.nanoTime();
+            long diff = TimeUnit.NANOSECONDS.toMillis((end - start));
+            log.info("Session run  timing in ms " + diff);
```

Comment:
Did we want to log this at info level?
I could imagine a busy web server having a lot of log spam...
`log.debug("Session run  timing in ms {}", diff);`
Same thing below.